pkg/ingester/metrics.go

```diff
@@ -303,7 +303,7 @@ func newIngesterMetrics(
 		activeSeriesPerUser: promauto.With(activeSeriesReg).NewGaugeVec(prometheus.GaugeOpts{
 			Name: "cortex_ingester_active_series",
 			Help: "Number of currently active series per user.",
-		}, []string{"user"}),
+		}, []string{"user", "attrib"}),
```

Comment:
To follow the pattern set by GEL, we would create a new metric instead of adding to the existing one. 

1. This new metric would be exposed on a separate endpoint apart from `/metrics` called `/usage_metrics`
2. The label _name_ should be dynamic and match the configured name, instead of using a constant `attrib` name

Reply:
Thanks, this is fixed in current PR.